src/System.IO.FileSystem.AccessControl/src/Resources/Strings.resx

```diff
@@ -141,7 +141,7 @@
   <data name="TypeUnrecognized_AccessControl" xml:space="preserve">
     <value>AccessControlType '{0}' unrecognized</value>
   </data>
-  <data name="InvalidOperation_AccessControl" xml:space="preserve">
-    <value>Invalid operation</value>
+  <data name="InvalidOperation_RemoveFail" xml:space="preserve">
+    <value>Remove fail</value>
```

Comment:
Perhaps be a more descriptive here: "Remove access rule failed"

Reply:
done